backoffice/src/Models/ViewModels/EmailView.cs

```diff
@@ -0,0 +1,26 @@
+using System.Xml.Schema;
+using Droits.Models.Domain;
+
+namespace Droits.Models;
+
+public class EmailView
+{
+    public EmailView()
+    {
+    }
+
+    public EmailView(Email email)
+    {
+        Id = email.Id;
+        Recipient = email.Recipient;
+        TypeOfEmail = email.GetType();
+        Date = email.DateLastModified;
+        Subject = email.Subject;
+    }
+    
+    public Guid Id { get; set; }
+    public string? Recipient { get; set; }
+    public Type TypeOfEmail { get; set; }
```

Comment:
This Type is a little too generic of naming, better to call it EmailType

Reply:
This may need elaboration (Type is currently a stand in atm)
Am I changing TypeOfEmail to EmailType?

Reply:
Ah yeah, so the enum against the Email should be EmailType, if it's just a stand-in make it a string or something. 

Think I got confused here that "Type" was the name of the type of email Enum